assets/js/frontend/give-donations.js

```diff
@@ -13,6 +13,25 @@ var give_scripts, give_global_vars;
 var Give = 'undefined' !== typeof Give ? Give : {};
 
 Give = {
+	init: function () {
+		var subHelperObjs = ['form'],
```

Comment:
@DevinWalker i added logic to main `init` function to initialise sub objects `init` function (if any). Also added two event. 

Please check event naming because we will following if you agree. Chosen is following same naming pattern.